File: BasicsWithJava/sharedPreferences/app/src/main/java/com/sumere/sharedpreferences/MainActivity.java
package com.sumere.sharedpreferences;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Context;
import android.content.SharedPreferences;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.EditText;
import android.widget.ImageView;

public class MainActivity extends AppCompatActivity {

    EditText num;
    EditText result;
    SharedPreferences sharedPreferences;
    int age = 0;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        num = findViewById(R.id.ageText);
        result = findViewById(R.id.resultText);
        sharedPreferences = this.getSharedPreferences("com.sumere.sharedpreferences", Context.MODE_PRIVATE);
        age = sharedPreferences.getInt("storedAge",-1);
        if(age != 0)
            result.setText(""+age);
    }

    public void saveAge(View view){
        if(!num.getText().toString().matches("")) {
            age = Integer.parseInt(num.getText().toString());
            result.setText("" + age);
            sharedPreferences.edit().putInt("storedAge",age).apply();
        }
    }

    public void delete(View view) {
        age = sharedPreferences.getInt("storedAge", -1);
        if (age != -1) {
            sharedPreferences.edit().remove("storedAge").apply();
            result.setText("");
            num.setText("");
        }
    }
}

File: KotlinCountries/app/src/main/java/com/sumere/kotlincountries/view/MainActivity.kt
package com.sumere.kotlincountries.view

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.view.LayoutInflater
import com.sumere.kotlincountries.R
import com.sumere.kotlincountries.databinding.ActivityMainBinding

class MainActivity : AppCompatActivity() {

    private lateinit var binding: ActivityMainBinding

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        binding = ActivityMainBinding.inflate(this@MainActivity.layoutInflater)
        setContentView(binding.root)
    }
}